servers/src/main/java/tachyon/worker/netty/NettyDataServer.java

```diff
@@ -73,9 +73,9 @@ private ServerBootstrap createBootstrap() {
     // set write buffer
     // this is the default, but its recommended to set it in case of change in future netty.
     boot.childOption(ChannelOption.WRITE_BUFFER_HIGH_WATER_MARK,
-        mTachyonConf.getInt(Constants.WORKER_NETTY_WATERMARK_HIGH, 32 * 1024));
+        (int)mTachyonConf.getBytes(Constants.WORKER_NETTY_WATERMARK_HIGH, 32 * 1024));
```

Comment:
It's weird that our style checker didn't catch this. Please create a JIRA for that.